packages/blaze/view.js

```diff
@@ -352,13 +352,16 @@ Blaze._materializeView = function (view, parentView, _workStack, _intoArray) {
       view._isInRender = false;
 
       if (! c.firstRun) {
+        // added by ramezrafla@zegenie to skip affecting DOM for unchanged content
+        if (Blaze._deepCompare(lastHtmljs,htmljs)) {
```

Comment:
No, it does not happen. So what I am saying is that you will never really do in your app:

```
{{value}}
```

Where:

```
value = {a: {b: 1}};
```

You might do:

```
{{value.a.b}}
```

But what I am saying is that in the comparison, Blaze just compares then values of `b`s, so primitive values.

So can you give me a realistic example where the value of `lastHtmljs` and `htmljs` is something else than what it is checked by `Blaze._isContentEqual`? I have [updated my example](https://github.com/mitar/test-blaze-array) with complex objects to test this.